Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -265,12 +256,9 @@ public final class DependencyTreeGenerator {
         // Create the mapping.
         let typeDescriptionToScopeMap: [TypeDescription: Scope] = reachableTypeDescriptions
             .reduce(into: [TypeDescription: Scope]()) { partialResult, typeDescription in
-                guard let instantiable = typeDescriptionToFulfillingInstantiableMap[typeDescription] else {
-                    // We can't find an instantiable for this type.
-                    // This is bad, but we handle this error in `validateReachableTypeDescriptions()`.
-                    return
```

Comment:
this was impossible to hit, and we already have `validateReachableTypeDescriptions()` to handle less egregious cases. So I folded in this `guard let` with the one below